Test failures with Python 3.13.0b1

- [x] I have marked all applicable categories:
    + [x] exception-raising bug
    + [ ] visual output bug
- [x] I have visited the [source website], and in particular
  read the [known issues]
- [x] I have searched through the [issue tracker] for duplicates
- [x] I have mentioned version numbers, operating system and
  environment, where applicable:
  ```python
  import tqdm, sys
  print(tqdm.__version__, sys.version, sys.platform)
  ```
  it outputs:
  ```
  4.66.4 3.13.0b1 (main, May 26 2024, 18:32:07) [GCC 15.0.0 20240519 (experimental)] linux
  ```

I tried to run tests for `tqdm-4.66.4` with recently released Python 3.13.0b1 in Gentoo and few have failed:
```
FAILED tests/tests_main.py::test_pipes - AssertionError: assert b'CODE_OF_CONDUCT.md\nCONTRIBUTING.md\nDEMO.ipynb\nLICENCE\nMakefile\nPKG-INFO\nREADME.rst\nenvironment.yml\nexamples\nlogo.png\npyproject.toml\nsetup.cfg\ntests\ntests_notebook.ipynb\ntox.ini\ntqdm\ntqdm.egg-info\n' == b''

  Full diff:
  - b''
  + (b'CODE_OF_CONDUCT.md\nCONTRIBUTING.md\nDEMO.ipynb\nLICENCE\nMakefile\nPKG-I'
  +  b'NFO\nREADME.rst\nenvironment.yml\nexamples\nlogo.png\npyproject.toml\nsetu'
  +  b'p.cfg\ntests\ntests_notebook.ipynb\ntox.ini\ntqdm\ntqdm.egg-info\n')
FAILED tests/tests_main.py::test_main_import - KeyError: 'iterable'
FAILED tests/tests_main.py::test_main_bytes - KeyError: 'iterable'
FAILED tests/tests_main.py::test_main_log - KeyError: 'iterable'
FAILED tests/tests_main.py::test_main - KeyError: 'iterable'
FAILED tests/tests_main.py::test_manpath - KeyError: 'iterable'
FAILED tests/tests_main.py::test_comppath - KeyError: 'iterable'
FAILED tests/tests_main.py::test_exceptions - KeyError: 'iterable'
```

--- a/tqdm/cli.py
+++ b/tqdm/cli.py
@@ -4,6 +4,7 @@ Module version for monitoring CLI pipes (`... | python -m tqdm | ...`).
 import logging
 import re
 import sys
+import textwrap
 from ast import literal_eval as numeric
 
 from .std import TqdmKeyError, TqdmTypeError, tqdm
@@ -177,7 +178,11 @@ def main(fp=sys.stderr, argv=None):
     logging.basicConfig(level=getattr(logging, logLevel),
                         format="%(levelname)s:%(module)s:%(lineno)d:%(message)s")
 
-    d = tqdm.__doc__ + CLI_EXTRA_DOC
+    d = tqdm.__doc__
+    if sys.version_info >= (3, 13):
+        # Python 3.13+ automatically dedents docstrings
+        d = textwrap.indent(d, "    ")
+    d += CLI_EXTRA_DOC
 
     opt_types = dict(RE_OPTS.findall(d))
     # opt_types['delim'] = 'chr'

--- a/tqdm/cli.py
+++ b/tqdm/cli.py
@@ -4,8 +4,8 @@ Module version for monitoring CLI pipes (`... | python -m tqdm | ...`).
 import logging
 import re
 import sys
-import textwrap
 from ast import literal_eval as numeric
+from textwrap import indent
 
 from .std import TqdmKeyError, TqdmTypeError, tqdm
 from .version import __version__
@@ -178,11 +178,9 @@ def main(fp=sys.stderr, argv=None):
     logging.basicConfig(level=getattr(logging, logLevel),
                         format="%(levelname)s:%(module)s:%(lineno)d:%(message)s")
 
-    d = tqdm.__doc__
-    if sys.version_info >= (3, 13):
-        # Python 3.13+ automatically dedents docstrings
-        d = textwrap.indent(d, "    ")
-    d += CLI_EXTRA_DOC
+    # py<3.13 doesn't dedent docstrings
+    d = (tqdm.__doc__ if sys.version_info < (3, 13)
+         else indent(tqdm.__doc__, "    ")) + CLI_EXTRA_DOC
 
     opt_types = dict(RE_OPTS.findall(d))
     # opt_types['delim'] = 'chr'

